app/_assets/javascripts/app.js

```diff
@@ -3,6 +3,8 @@
 'use strict'
 
 $(function () {
+  var NAV_HEIGHT = 56;
```

Comment:
I'm not sure why there were so many different values for different scrollTop but I standardized it to 56px (the current height). If this is incorrect, hopefully, some light could be shed on why the other, seemingly, arbitrary values were picked?